invenio_app_ils/documents/mappings/os-v1/documents/document-v2.0.0.json

```diff
@@ -891,7 +985,14 @@
             "type": "keyword"
           },
           "value": {
-            "type": "keyword"
+            "type": "keyword",
+            "fields": {
```

Comment:
I think it will be unlikely to search by URL, we could remove this part, @ntarocco WDYT?